src/Navigation/Xamarin.Forms/Models/Registration/NavigationRegistrationTT.cs

```diff
@@ -0,0 +1,18 @@
+﻿using CodeMonkeys.Core.Interfaces.MVVM;
```

Comment:
Don't know if I really like the naming of the file here. The name is saying nothing except the count of generic arguments. Maybe we should discuss this?

Reply:
Feel free to suggest a different naming pattern.
I just sticked to our coding conventions.
I doubt we will find a pattern that satisfies both the parameter count and there intention without having super long file names.